runtimes/peregrine/src/lib.rs

```diff
@@ -227,6 +227,7 @@ impl pallet_transaction_payment::Config for Runtime {
 impl pallet_sudo::Config for Runtime {
 	type RuntimeCall = RuntimeCall;
 	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = ();
```

Comment:
This should use the sudo weights instead, no?

Reply:
Updated in: [4d040ad](https://github.com/KILTprotocol/kilt-node/pull/541/commits/4d040ad59ea32b76034ee7bb32cc4874789ed243)